saviSimulation/savi/jason_processing/UAS.java

```diff
@@ -89,6 +96,28 @@ public void draw() {
 		
 		
 	}
-	
+
+	@Override
+	public  List<String> getOutgoingMessages(){
+		
+		List<String> myMsgOutCopy = new LinkedList<String>();
+		myMsgOutCopy.addAll(getBehavior().agentState.getMsgOutAll());
```

Comment:
This line feels a little convoluted, but I think I understand why. I see that the linkage to the agentState lives in the behaviour. Is there any way we can simplify this?